Sources/Commands/Workspace.swift

```diff
@@ -512,7 +515,7 @@ public class Workspace {
                 // FIXME: Issue suitable diagnostics for cases where an
                 // update is needed, or cases where the range is invalid.
                 fatalError("unexpected dependency resolution result")
-            case .removed: break //FIXME: TODO
+            case .removed: try remove(specifier: specifier)
```

Comment:
OT for this patch, but I wonder how we should handle errors that occur during this process. Should we try and only "commit" the changes after we have validated everything, or should we simply be resilient to the workspace potentially having been left in an inconsistent state?